src/net/http/http.android.ts

```diff
@@ -43,17 +44,20 @@ export default class HttpAndroid extends NativeMobileComponent<any, MobileOSProp
   private _client: any;
   constructor(params?: Partial<IHttp>) {
     super(params);
-    this.timeout = 60000;
+    this.postConstruct();
   }
 
   protected preConstruct(params?: Partial<Record<string, any>>): void {
     this._clientBuilder = new OkHttpClientBuilder();
-    this._client = this._clientBuilder.build();
-
+    this.timeout = DEFAULT_TIMEOUT;
     _instanceCollection.push(this);
     super.preConstruct(params);
   }
 
+  protected postConstruct() {
```

Comment:
Postconstruct doesn't need to be an extra function. Simply write the line after constructor super.